pkg/sloop/webserver/webserver.go

```diff
@@ -167,30 +171,45 @@ func healthHandler() http.HandlerFunc {
 	}
 }
 
-func Run(config WebConfig, tables typed.Tables) error {
-	webFiles = config.WebFilesPath
-	server := &Server{}
-	server.mux = http.NewServeMux()
-	server.mux.HandleFunc("/webfiles/", webFileHandler)
-	server.mux.HandleFunc("/data/backup", backupHandler(tables.Db(), config.CurrentContext))
-	server.mux.HandleFunc("/data", queryHandler(tables, config.MaxLookback))
-	server.mux.HandleFunc("/resource", resourceHandler(config.ResourceLinks))
+// Handler for redirecting / to /currentContext to ensure backward compatibility
+func redirectHandler(currentContext string) http.HandlerFunc {
+	return func(writer http.ResponseWriter, request *http.Request) {
+		redirectURL := "/"+currentContext
+		http.Redirect(writer, request, redirectURL, http.StatusTemporaryRedirect)
+	}
+}
+
+// Registers paths for mux router
+func registerPaths(router *mux.Router, config WebConfig, tables typed.Tables) {
+	router.PathPrefix("/webfiles/").HandlerFunc(webFileHandler(config.CurrentContext))
```

Comment:
any reason this line use router.PathPrefix, while the rest use router.HandleFunc, can we all use router.HandleFunc

Reply:
Yes that is because the full path of static files will be a whole path of it's own for example /webfiles/index.html. In this case all requests targeting static files should target webfiles handler. This works by using the prefix router method.
For details https://github.com/gorilla/mux#static-files